### ISSUE TITLE
get a random paramter value by default to avoid collision

### ISSUE THREAD

**Issue by OskarStark** [Incenteev/ParameterHandler#78]
In our company, we use the following setup.

A development server where different developers work on different or same projects.

We have a symfony application which uses elastic search (one elastic search server for all developers)
Every working copy has its own index in `elasticsearch_index_name`

for now we use a convention like: `PROJECT-NAME_DEVELOPER-NAME`, so the index of my working copy is named:
`elasticsearch_index_name: SYMFONY_OSKARSTARK`

I can't write this to my parameters.yml.dist:
`elasticsearch_index_name: SYMFONY`

But in my case the name is irrelevant (a hash would be great, or a timestamp), is there a possibility to generate a random string or a timestamp to avoid this collision, if a developer just enter through the interactive questions for the parameters?

I hope you understand my problem!

Best regards Oskar

**Comment by Ninir** [comment]
@OskarStark Hey!

I achieved another related subject by adding a custom Composer\ScriptHandler, called on the `pre-install-cmd` event.
I wanted to generate a given string on composer install, without overriding the actual value of the parameter.

This is the static method from the custom composer handler:

```php
public static function generateAdminPrefix(CommandEvent $event)
{
   $options = self::getOptions($event);
   $file = $options['incenteev-parameters']['file'];

   if (is_file($file)) {
       return;
   }

   putenv('SF2_TEST=' . md5(uniqid(mt_rand(), true)));
}
```

Then, I added to the composer.json env-map:

```json
"test": "SF2_TEST"
```

When being generated, the `parameters.yml` file now gets the environment variable, adding it as default value for the specified parameter.
